compiler-core/src/warning.rs

```diff
@@ -50,7 +43,8 @@ your program.",
 
                 type_::Warning::ImplicitlyDiscardedResult { location } => Diagnostic {
                     title: "Unused result value".into(),
-                    text: "Hint: If you are sure you don't need it you can assign it to `_`".into(),
+                    text: "".into(),
+                    hint: Some("If you are sure you don't need it you can assign it to `_`".into()),
```

Comment:
Are there any tests for this error formatting? I think this one has changed the formatting to have 3 empty lines in a row

Reply:
It's not writing 3 lines, but I'll provide the tests anyway, thanks.

Reply:
I'll provide the tests for this

Reply:
It's a warning and we doesn't have anything to snapshot warnings. The output is going like this:
![image](https://user-images.githubusercontent.com/42675056/157879482-ff1fd29f-0bb0-4f38-b940-c5ba0228269d.png)

Reply:
I've created a snapshot test for warnings. I think now your comment is covered